pulp_2to3_migration/app/pre_migration.py

```diff
@@ -288,7 +288,7 @@
     mongo_importer_q = mongo_Q(repo_id=repo.pulp2_repo_id)
 
     # importers with empty config are not needed - nothing to migrate
-    mongo_importer_q &= mongo_Q(config__ne={})
+    mongo_importer_q &= mongo_Q(config__ne={}) & mongo_Q(config__exists=True)
```

Comment:
tiny suggestion to switch the order, if config does not exist, second part wont be executed

Reply:
really? I thoughy mongo doesn't have lazy evaluation.
I'll change, just in case.